src/condor_credd/condor_credmon_oauth/credmon/CredentialMonitors/LocalCredmon.py

```diff
@@ -58,7 +62,16 @@ def refresh_access_token(self, username, token_name):
         token.update_claims({'sub': username})
         user_authz = self.authz_template.format(username=username)
         token.update_claims({'scope': user_authz})
-        token.update_claims({'ver': 'scitokens:2.0'})
+
+        # Only set the version if we have one.  No version is valid, and implies scitokens:1.0
```

Comment:
`No version is valid`: Does an empty string imply `scitokens:1.0`? I believe `LOCAL_CREDMON_TOKEN_VERSION=` will be interpreted as `""` and not `None`.

Reply:
I think "" will still test as false in Python?

Reply:
Ah, right, carry on :)